torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -454,16 +486,7 @@ def _need_skip_config(
         ):
             warnings.warn("Mixed QAT and Non-QAT quantization config is not supported.")
             need_skip = True
-        if current_mode.dynamic_state is not None:
-            input_activation_spec = quantization_config.input_activation
-            if (
-                input_activation_spec is not None
-                and current_mode.dynamic_state != input_activation_spec.is_dynamic
-            ):
-                warnings.warn(
-                    "Mixed dynamic and static quantization config is not supported."
-                )
-                need_skip = True
```

Comment:
Because we always set `is_dynamic=False` for `linear_dynamic_fp16` but it needs to work with dynamic quantization of other ops. Besides, I didn't see an issue if users use mixed static/dynamic quantization for different ops.

Reply:
Thanks. I have reverted this change because this is not an issue for now